aggregate-snapshot/aggregate-snapshot-repository/src/test/java/uk/gov/justice/services/eventsourcing/jdbc/snapshot/SnapshotJdbcRepositoryJdbcIT.java

```diff
@@ -54,14 +65,27 @@ public void shouldStoreAndRetrieveSnapshot() {
         final UUID streamId = randomUUID();
         final AggregateSnapshot aggregateSnapshot = createSnapshot(streamId, VERSION_ID, TYPE, AGGREGATE);
 
-        snapshotJdbcRepository.storeSnapshot(aggregateSnapshot);
+        final boolean returnValue = snapshotJdbcRepository.storeSnapshot(aggregateSnapshot);
```

Comment:
I know this is only a test, but the name 'returnValue' doesn't help with understanding what happened. Maybe a more informative name would help here